AngouriMath/Convenience/MathS.cs

```diff
@@ -536,7 +526,7 @@ public static class Settings
             /// <summary>
             /// Numbers whose absolute value is less than PrecisionErrorZeroRange are considered zeros
             /// </summary>
-            public static Setting<EDecimal> PrecisionErrorZeroRange { get; } = EDecimal.FromDecimal(1.0e-16m);
+            public static Setting<EDecimal> PrecisionErrorZeroRange { get; } = EDecimal.FromDecimal(1e-15m);
```

Comment:
What made you change the default precision?

Reply:
Ah yes, forgot to revert this.